Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.cpp

```diff
@@ -1513,11 +1513,42 @@ bool Scene_surface_mesh_item::hasPatchIds()
 bool
 Scene_surface_mesh_item::save(std::ostream& out) const
 {
+  std::vector<std::string> internal_properties;
+  std::vector<std::string> vprop = d->smesh_->properties<vertex_descriptor>();
+  std::vector<std::string> fprop = d->smesh_->properties<face_descriptor>();
+  for(auto s : vprop)
+     if (s.compare("v:normal") == 0
+         || s.compare("v:texcoord") == 0
+         || s.compare("v:color") == 0)
+    {
+       internal_properties.push_back(s);
```

Comment:
You can fill message directly here and use a user friendly text like "Vertex Normals"...